### PR TITLE
Emmet branch

### PR DESCRIPTION
**Pull request by l00144427**
Added a starting point for the social contract, changed the Software section slightly & defined the Scrum Master role

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,4 +98,67 @@ Please see our list of Risks
...
+- Be honest.
+
+**Communication**
+
+- Slack is the preferred method of communication.
+- If a demonstration is required use Zoom, record the session and upload to the Slack channel.
+- No Slack communications between midnight and 7am GMT.
+- Raise a problem as soon as you see it.
+- Respect each other and understand differences in knowledge.
+- Respect each other and understand differences in knowledge.
```
Comment (cjfarrelly) [inline README.md]: same thing twice

**Review by cjfarrelly — APPROVED** [review]

**Review by rlennon — CHANGES_REQUESTED** [review]
Missing statement: 
Keep your language and tone professional at all times
It may seem obvious but it needs to be included.

File: README.md (hunk 1 above)
Comment (l00144427) [reply README.md]: I removed that extra line.

**Review by ghost — APPROVED** [review]
looks good!

**Review by rlennon — APPROVED** [review]

**Pull request merged**